cmd/webhook/main.go

```diff
@@ -46,10 +48,17 @@ func main() {
 	flag.StringVar(&admissionListenAddress, "listen-address", ":9876", "The address on which the MutatingWebhook will listen on")
 	flag.StringVar(&admissionTLSCertPath, "tls-cert-path", "/tmp/cert/cert.pem", "The path of the TLS cert for the MutatingWebhook")
 	flag.StringVar(&admissionTLSKeyPath, "tls-key-path", "/tmp/cert/key.pem", "The path of the TLS key for the MutatingWebhook")
+	flag.StringVar(&caBundleFile, "ca-bundle", "", "(optional) path to a file containing all PEM-encoded CA certificates (will be used instead of the host's certificates)")
```

Comment:
Nit: Just to follow the pattern as the previous flags(I don't think the `optional` hint)
```suggestion
	flag.StringVar(&caBundleFile, "ca-bundle", "", "Path to a file containing all PEM-encoded CA certificates (will be used instead of the host's certificates if set)")
```